src/dmd/cppmangle.d

```diff
@@ -761,6 +761,14 @@ public:
         buf.writeByte(c);
     }
 
+    override void visit(TypeNull t)
+    {
+        if (t.isImmutable() || t.isShared())
+            return error(t);
```

Comment:
Not sure how well `immutable`/`shared` is covered in the tests for other types but this looks like it's not covered for this type.

Reply:
Correct. I matched the precedent.